src/components/Messages/ChatroomPreview.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
```

Comment:
I'd install the [prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) extension to automagically format your code on save. It helps add to the readability, and follows a specific styleguide across a repository